db/models.js

```diff
@@ -20,8 +20,18 @@ const fetchReviews = () => {
         .then (({rows}) => rows)
     }
 
+const fetchReviewByID = (sort_by) => {
+    const search = sort_by
+    const sqlString = `SELECT * FROM reviews
+    WHERE review_id=${search};`
```

Comment:
This is open to SQL Injection, use the $1 method and give the query and array with the ID